MainWindow.xaml.cs

```diff
@@ -515,7 +567,7 @@ private void EnsureDiscInformation()
             }
 
             // Special case for Custom input
-            if (tuple.Item1 == "Custom Input" && tuple.Item2 == KnownSystem.NONE && tuple.Item3 == DiscType.NONE)
+            if (isCustom && selectedSystem == KnownSystem.NONE && selectedDiscType == DiscType.NONE)
```

Comment:
If you're making `isCustom` its own variable, why not wrap in the other two parts of this conditional into that one boolean?

Reply:
Actually I don't like `isCustom` neither, I'll just define a new `KnownSystem.CUSTOM` for this.